docs/console.md

```
@@ -0,0 +1,129 @@
+| `MODIFIED` | The file has been changed since the last scaffold run.         |
+| `missing`  | The file was written by scaffold but no longer exists on disk. |
+
+## yii scaffold/diff `<file>`
+
+Shows a line-by-line diff between the provider stub and the current on-disk file.
+
+```bash
+yii scaffold/diff config/params.php
+```
+
+Lines present only in the stub are prefixed with `- `, lines present only in the current file
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)